### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by Tonksthebear**
https://github.com/department-of-veterans-affairs/va-mobile-app/issues/8209
## Description of Change
Updates documentation with new found maintenance window details

## Testing

- [ ] Tested on iOS 
- [ ] Tested on Android 

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [ ] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/documentation/docs/Engineering/BackEnd/Features/MaintenanceWindows.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Maintenance windows are periods of time during which specific vets-api functiona
...
+Unlike most of the data used in the vets-api engine, `MaintenanceWindows` come from the vets-api database and not from upstream servers. They contain the external service name, start time, and end time. They're added there by developers on other teams to PagerDuty, and a Sidekiq job populates the database based on PagerDuty. There is a [white list](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/9864e007626ceb2cd5c92b8c8838af1dbb3b4b7d/apps/vets-api/prod/values.yaml#L2169) of maintenance windows on GitHub that the sidekiq job uses to connect to PagerDuty and populate the maintenance windows. The mobile app requests maintenance windows via the maintenance windows controller, which searches the database for `MaintenanceWindow` records that end in the future and creates a `ServiceGraph` object to process those windows into a format that is more easily used by the front-end.
```
Comment (kpethtel) [inline VAMobile/documentation/docs/Engineering/BackEnd/Features/MaintenanceWindows.md:9]: This is a good addition. I think the sentence starting "They're added there by developers on other teams to PagerDuty" might be more clearly worded as "Maintenance windows are added to PagerDuty by other teams, and a vets-api sidekiq job polls PagerDuty and populates MaintenanceWindow database records with the response."

**Review by kpethtel — DISMISSED** [review]
Looks good. One editorial suggestion.

**Review by aherzberg — APPROVED** [review]
LGTM

File: VAMobile/documentation/docs/Engineering/BackEnd/Features/MaintenanceWindows.md (hunk 1 above)
Comment (Tonksthebear) [reply VAMobile/documentation/docs/Engineering/BackEnd/Features/MaintenanceWindows.md]: Ah ya that wording is much clearer

**Review by kpethtel — APPROVED** [review]

**Pull request merged**